.github/workflows/build.yml

```diff
@@ -21,6 +21,25 @@ jobs:
       - name: Run Biome
         run: biome ci .
 
+  check-yarn-lock:
```

Comment:
@daniellockyer I thought you might be interested in this, and wondered if you have any feedback on the approach